app-text/mupdf/files/mupdf-1.20.0-lcms2.patch

```diff
@@ -0,0 +1,20 @@
+--- a/thirdparty/lcms2/src/cmsplugin.c
++++ b/thirdparty/lcms2/src/cmsplugin.c
+@@ -177,7 +177,7 @@ cmsBool CMSEXPORT  _cmsReadFloat32Number(cmsContext ContextID, cmsIOHANDLER* io,
+     if (n != NULL) {
+ 
+         tmp = _cmsAdjustEndianess32(tmp);
+-        *n = *(cmsFloat32Number*)(void*)&tmp;
++        memcpy(&n, &tmp, sizeof tmp);
```

Comment:
Since `memcpy()` is used, this patch should also add `#include <string.h>`.

Reply:
Yep, looks like it's supposed to be that way.

Reply:
please make this patch more solid by putting a
```
#if __STDC_VERSION__ >= 201112L
static_assert(sizeof(*n) == sizeof(*tmp), "*n and *tmp have different sizes");
#endif
```
and `#include <assert.h>`

That way, any change of types will immediately cause a compile-time failure.

Reply:
```suggestion
+        memcpy(n, &tmp, sizeof tmp);
```
because `n` is a pointer already?

Reply:
Thanks, fixed.